packages/amplify_authenticator/lib/src/state/auth_viewmodel.dart

```diff
@@ -19,19 +19,31 @@ import 'package:amplify_authenticator/src/blocs/auth/auth_bloc.dart';
 import 'package:amplify_authenticator/src/blocs/auth/auth_data.dart';
 import 'package:flutter/material.dart';
 
+// a list of screen transitions to NOT clear form data for
+const List<_ScreenTransition> _skipClearList = [
```

Comment:
This list probably needs to be added to. These two are needed for the typical sign up -> confirm sign up -> sign in flow.